criu/cr-service.c

```diff
@@ -397,6 +411,42 @@ static int setup_opts_from_req(int sk, CriuOpts *req)
 		goto err;
 	}
 
+	/*
+	 * Parse only the options relevant to the CHECK workflow:
+	 * log_file, log_to_stderr, and log_level.
+	 *
+	 * If logging to a file, either explicitly specified with log_file
+	 * or when using the DEFAULT_LOG_FILENAME, also resolve the
+	 * images_dir and work_dir.
+	 */
+	if (opts.mode == CR_CHECK) {
```

Comment:
yeah. Last time, when I said that we didn't need the separate parser for CR_CHECK, I mean the current parser should work for this command and any other commands. Maybe I don't understand something, but the commit message doesn't make it clear to me either. I know that some option are not applicable to the check command and users should not user them, but there is nothing wrong still processing them. As for image_dir, we need to open it only when it is specified. It is ok to check the `check` command in resolve_images_dir_path.